tests/mlmodel_openai/test_chat_completion.py

```diff
@@ -283,6 +284,25 @@ def test_openai_chat_completion_sync_in_txn_no_llm_metadata(set_trace_info):
     )
 
 
+@reset_core_stats_engine()
+@disabled_ai_monitoring_streaming_settings
+@validate_custom_events(chat_completion_recorded_events_no_llm_metadata)
+# One summary event, one system message, one user message, and one response message from the assistant
+@validate_custom_event_count(count=4)
+@validate_transaction_metrics(
+    "test_chat_completion:test_openai_chat_completion_sync_in_txn_no_llm_metadata_stream_monitoring_disabled",
+    scoped_metrics=[("Llm/completion/OpenAI/create", 1)],
+    rollup_metrics=[("Llm/completion/OpenAI/create", 1)],
+    background_task=True,
+)
+@background_task()
+def test_openai_chat_completion_sync_in_txn_no_llm_metadata_stream_monitoring_disabled(set_trace_info):
```

Comment:
I realize some of our test names are getting really long and the names are implying we are trying to test multiple things in a single test. Given we aren't testing a case where LLM metadata is present when streaming is disabled, we could probably drop the "`no_llm_metadata`" part in the test name.